README.md

```diff
@@ -25,6 +25,17 @@ If you want that, use a markup processor before running the sanitizer, like [pul
 [pulldown-cmark]: https://github.com/google/pulldown-cmark
 
 
+Installation
+-----------
+
+To use `ammonia`, add this to your `[dependencies]` section:
```

Comment:
Does `[dependencies]` show up twice on the rendered page? How about:

To use `ammonia`, add it to your project's `Cargo.toml` file: